data/travis/setup.sh

```diff
@@ -57,19 +58,12 @@ else
   distribution=$(lsb_release --short --id | tr 'A-Z' 'a-z')
   case $distribution in
     debian)
-      code_name=$(lsb_release --short --codename)
-      component=main
-      apt_url_base=https://packages.groonga.org
-      cat <<EOF | sudo tee /etc/apt/sources.list.d/groonga.list
-deb ${apt_url_base}/${distribution}/ ${code_name} ${component}
-deb-src ${apt_url_base}/${distribution}/ ${code_name} ${component}
-EOF
-      sudo apt-get update -qq
-      sudo apt-get install -qq -y --allow-unauthenticated groonga-keyring
+      wget https://packages.groonga.org/debian/groonga-apt-source-latest-$(lsb_release --codename --short).deb
```

Comment:
Can we use `$(lsb_release --id --short | tr 'A-Z' 'a-z')` for the `debian`/`ubuntu` part?

Reply:
Or can we use `${distribution}`?